examples/sampling/graphbolt/link_prediction.py

```diff
@@ -175,7 +175,9 @@ def create_dataloader(args, graph, features, itemset, is_train=True):
     # [Role]:
     # Initialize a neighbor sampler for sampling the neighborhoods of nodes.
     ############################################################################
-    datapipe = datapipe.sample_neighbor(graph, args.fanout)
+    datapipe = datapipe.sample_neighbor(
+        graph, args.fanout, True if args.output_cscformat == "True" else False
```

Comment:
Please always specify the **Key Word** for `kwargs` or it's confusing and error-prone. In this case, are you specifying `replace=xxx` or `output_cscformat`? https://github.com/dmlc/dgl/blob/de14619cdeed5f77c6935f5267049f1d9f764f43/python/dgl/graphbolt/impl/neighbor_sampler.py#L103